docker_readme.txt

```diff
@@ -0,0 +1,9 @@
+Steps for Docker: 
+ 1. install docker or docker for windows 
```

Comment:
which docker version should I use? Any will do?

Reply:
??

Reply:
AFAIK, yes, I don't see a reason to state a specific version.